core/src/main/scala/org/apache/spark/util/AccumulatorV2.scala

```diff
@@ -112,7 +112,18 @@ abstract class AccumulatorV2[IN, OUT] extends Serializable {
    * Creates a new copy of this accumulator, which is zero value. i.e. call `isZero` on the copy
    * must return true.
    */
-  def copyAndReset(): AccumulatorV2[IN, OUT]
+  def copyAndReset(): AccumulatorV2[IN, OUT] = copy().reset()
+
+  /**
+   * Creates a new copy of this accumulator.
+   */
+  def copy(): AccumulatorV2[IN, OUT]
+
+  /**
+   * Resets this accumulator, which is zero value. i.e. call `isZero` must
+   * return true.
+   */
+  def reset(): AccumulatorV2[IN, OUT]
```

Comment:
I know this can save some line of code, but I prefer to define it as `def reset(): Unit`, which matches its semantic better.